CHANGELOG.md

```diff
@@ -5,9 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
 ### Added
 - Remove duplicates in the sub-event array in super events with back end check. [#834](https://github.com/rokwire/events-manager/issues/834)
 - Add "Back To Super Event" button for sub events. [#853](https://github.com/rokwire/events-manager/issues/853)
+- Add rich editor for user event description. [#721](https://github.com/rokwire/events-manager/issues/721)
 
 ### Fixed
 - Fix time filters between campus events and user events. [#846](https://github.com/rokwire/events-manager/issues/846)
```

Comment:
This entry can be removed. It's present at the bottom of this section and that's the correct place as changes are sequential. Thanks.